src/state/slices/tradeRampInputSlice/tradeRampInputSlice.ts

```diff
@@ -0,0 +1,86 @@
+import type { PayloadAction } from '@reduxjs/toolkit'
+import type { AccountId, ChainId } from '@shapeshiftoss/caip'
+import { btcAssetId, ethAssetId } from '@shapeshiftoss/caip'
+import type { Asset } from 'packages/types/src/base'
+
+import { defaultAsset } from '../assetsSlice/assetsSlice'
+import type { BaseReducers } from '../common/tradeInputBase/createTradeInputBaseSlice'
+import { createTradeInputBaseSlice } from '../common/tradeInputBase/createTradeInputBaseSlice'
+
+import type { FiatCurrencyItem, RampQuote } from '@/components/Modals/FiatRamps/config'
+import { localAssetData } from '@/lib/asset-service'
+import type { TradeInputState } from '@/state/slices/tradeInputSlice/tradeInputSlice'
+
+export type TradeRampInputState = {
+  buyFiatCurrency: FiatCurrencyItem | undefined
+  sellFiatCurrency: FiatCurrencyItem | undefined
+  slippagePreferencePercentage: string | undefined
+  buyAsset: Asset
+  sellAsset: Asset
+  sellAccountId: AccountId | undefined
+  buyAccountId: AccountId | undefined
+  sellAmountCryptoPrecision: string
+  sellFiatAmount: string
+  isInputtingFiatSellAmount: boolean
+  manualReceiveAddress: string | undefined
+  isManualReceiveAddressValidating: boolean
+  isManualReceiveAddressEditing: boolean
+  isManualReceiveAddressValid: boolean | undefined
+  selectedSellAssetChainId: ChainId | 'All'
+  selectedBuyAssetChainId: ChainId | 'All'
+  selectedFiatRampQuote: RampQuote | null
+}
+
+const initialState: TradeRampInputState = {
+  buyAsset: localAssetData[btcAssetId] ?? defaultAsset,
+  sellAsset: localAssetData[ethAssetId] ?? defaultAsset,
+  buyFiatCurrency: undefined,
+  sellFiatCurrency: undefined,
+  sellAccountId: undefined,
+  buyAccountId: undefined,
+  sellAmountCryptoPrecision: '0',
+  sellFiatAmount: '0',
+  isInputtingFiatSellAmount: false,
+  manualReceiveAddress: undefined,
+  isManualReceiveAddressValidating: false,
+  isManualReceiveAddressValid: undefined,
+  isManualReceiveAddressEditing: false,
+  slippagePreferencePercentage: undefined,
+  selectedSellAssetChainId: 'All',
+  selectedBuyAssetChainId: 'All',
+  selectedFiatRampQuote: null,
+}
+
+export const tradeRampInput = createTradeInputBaseSlice({
+  name: 'tradeRampInput',
+  initialState,
+  extraReducers: (_baseReducers: BaseReducers<TradeRampInputState>) => ({
+    setSellAssetUtxoChangeAddress: (
```

Comment:
This is fluff here?

Reply:
Removed in [622b3b2](https://github.com/shapeshift/web/pull/10592/commits/622b3b2a98efda4bd32ce27229e2d08fcceff629)